feature-group-definitions/array-flat.yml

```diff
@@ -1,3 +1,4 @@
+name: "Array.prototype.flat and Array.prototype.flatMap"
```

Comment:
Following the pattern of "Array at()":

```suggestion
name: "Array flat() and flatMap()"
```

Reply:
I like this last pattern. A template for single JS and Web API bits like this might be "Interface methodname/propertyname" in the absence of more widely used names.

(Apologies for the weekend commenting—not expecting a quick turnaround here. I've had a messy week, personally.)